x/pairing/types/expected_keepers.go

```diff
@@ -67,6 +67,6 @@ type BankKeeper interface {
 
 type ProjectsKeeper interface {
 	GetProjectDevelopersPolicy(ctx sdk.Context, developerKey string, blockHeight uint64) (policy projectstypes.Policy, err error)
-	AddComputeUnitsToProject(ctx sdk.Context, developerKey string, blockHeight uint64, cu uint64) (err error)
```

Comment:
orenS is in the middle of using/changing this function and this can lead to a problem when he does his PR, lets keep the renaming to another PR or let him do it